### ISSUE TITLE
Optimize fastfloat_strncasecmp

### ISSUE THREAD

**Issue by lemire** [fastfloat/fast_float#60]
These functions can be specialized for fixed length and done with SWAR

**Comment by IRainman** [comment]
Yes, it's definitely needed because fastfloat_strncasecmp is only supports some printable ASCII chars https://en.wikipedia.org/wiki/ASCII#Printable_character_table
